server/webapp/WEB-INF/rails.new/app/helpers/application_helper.rb

```diff
@@ -53,10 +53,10 @@ def url_for_job(job)
 
   def path_for_stage(stage_identifier)
     stage_identifier = stage_identifier.getIdentifier() if stage_identifier.respond_to? :getIdentifier
-    stage_detail_tab_path :pipeline_name => stage_identifier.getPipelineName(),
-                      :pipeline_counter => stage_identifier.getPipelineCounter(),
-                      :stage_name => stage_identifier.getStageName(),
-                      :stage_counter => stage_identifier.getStageCounter()
+    stage_detail_tab_path pipeline_name:    stage_identifier.getPipelineName(),
```

Comment:
Unnecessary spacing detected.<br>Do not use parentheses for method calls with no arguments.